mycpp/gc_list_test.cc

```diff
@@ -408,6 +408,41 @@ TEST contains_test() {
   PASS();
 }
 
+TEST test_list_sort() {
+  auto s1 = StrFromC("fooA");
+  auto s2 = StrFromC("fooB");
+  auto s3 = StrFromC("fooC");
+  auto l = NewList<Str*>(std::initializer_list<Str*>{s3, s1, s2});
+
+  auto s = sorted(l);
+  ASSERT(str_equals(s->index_(0), s1));
+  ASSERT(str_equals(s->index_(1), s2));
+  ASSERT(str_equals(s->index_(2), s3));
+
+  PASS();
+}
+
+TEST test_list_remove() {
+  auto l = NewList<int>(std::initializer_list<int>{1, 3, 3, 3, 2});
+
+  for (int i = 0; i < 3; ++i) {
+    l->remove(3);
+  }
+
+  bool caught = false;
+  try {
+    l->index(3);
+  } catch (ValueError* e) {
+    caught = true;
+  }
+  ASSERT(caught);
+
+  ASSERT_EQ(l->index_(0), 1);
```

Comment:
Generally the style for these types of frameworks is to put the expected value FIRST

If you do `ASSERT_EQ_FMT()` then you'll see `greatest.h` does respect that convention

i.e. it prints a message "Expected: arg1, Actual: arg2"

For these cases it's not a big deal

Reply:
Ah whoops. I'm used to gtest which encourages the opposite order (At least in recent years. I think it used to be the other way around). Will send a followup to make these consistent with the rest